iron-multi-selectable.html

```diff
@@ -12,7 +12,15 @@
 <link rel="import" href="iron-selectable.html">
 
 <script>
-  /** @polymerBehavior Polymer.IronMultiSelectableBehavior */
+  /**
+   * ### Notable breaking changes between 1.x and 2.x (hybrid):
```

Comment:
This is here so that it shows up in the docs shown by iron-component-page when you switch to `IronSelectableBehavior` or `IronMultiSelectableBehavior`, but I think adding a reference to it in the README would be good.